_inc/client/pro-status/index.jsx

```diff
@@ -212,12 +213,12 @@ class ProStatus extends React.Component {
 				}
 			}
 
-			if ( sitePlan.product_slug ) {
-				if ( ! hasFree ) {
-					if ( active && installed ) {
-						return this.getProActions( 'active' );
-					}
+			if ( sitePlan.product_slug && ! hasFree ) {
+				if ( active && installed ) {
+					return this.getProActions( 'active' );
+				}
 
+				if ( includes( [ 'akismet', 'backups', 'rewind', 'scan' ], feature ) ) {
```

Comment:
I feel like this just patches the fact that we're parsing non-plugins as plugins here. `installed` and `active` as used in this `if` block are only meaningful in the context of plugins . I pushed #9096 a few mins ago, with another approach for the same issue.